src/core/namespace.c

```diff
@@ -309,6 +331,7 @@ static void drop_duplicates(BindMount *m, unsigned *n) {
                  * above. */
                 if (previous && path_equal(f->path, previous->path)) {
                         log_debug("%s is duplicate.", f->path);
+                        free(f->path);
```

Comment:
Hm, even though this free *is* correct, it looks dangerous. Maybe `f->path = mfree(f->path)` would be safer.